developing.md

```diff
@@ -72,30 +72,3 @@ $ yarn test
 > When running tests, code changes don't need to be built with `yarn build` first because the test suite uses ts-node as its runtime environment. Otherwise, run `yarn build` before manually testing changes.
 
 <br />
```

Comment:
I think this should supplant contributing/developing.md

Reply:
Consolidated and now just have the one inside contributing folder